src/guides/v2.2/design-styleguide/errors-and-messages/errors-and-messages.md

```diff
@@ -2,42 +2,37 @@
 group: admin-style-guide
 subgroup: Writing and Content
 title: Admin Errors and Messages Writing
-menu_title: Errors and Messages Writing
-menu_order: 2
-menu_node:
 ---
 Magento uses different types of dynamic messaging to alert users when:
 
-*  An error has occurred and blocks them in completing a task
+*  An error has occurred and blocks them from completing a task.
 
-*  Users need to know important information before continuing a task
+*  Users need to know important information before continuing a task.
 
-## Types of Errors and Messages
+## Types of errors and messages
 
-*  **Information information.** This messaging is not an error. It's used when a person can continue with the task but needs to be alerted regarding information critical to completing a task.
+*  **Informational**: This messaging is not an error. It is used when a person can continue with the task but needs to be alerted regarding information critical to completing the task.
```

Comment:
```suggestion
*  **Informational**: This messaging is not an error. It is used when a person can continue with the task but needs to be alerted about information necessary to complete the task.
```

Ignore this suggestion if you want--I'm against using the word "critical" in an informational message.

Reply:
Nice distinction. "Critical" could be construed as a bit alarmist.